src/rt-app_utils.h

```diff
@@ -51,7 +51,7 @@ do {                                                                    \
     if (level <= LOG_LEVEL) {                                           \
         struct timespec t_now;                                          \
         clock_gettime(CLOCK_MONOTONIC, &t_now);                         \
-        fprintf(where, LOG_PREFIX level_pfx " %06ld.%09ld " msg "\n", t_now.tv_sec, t_now.tv_nsec, ##args);		\
+        fprintf(where, LOG_PREFIX "[%06ld.%09ld] " level_pfx msg "\n", t_now.tv_sec, t_now.tv_nsec, ##args);		\
```

Comment:
Why do you want to change the log format ?
Would be good to give the reason in commit message

Reply:
This is a change Juri made - @jlelli can you remember why you wanted to do this? I just rolled it up as part of the original commit in the latest rebase.

Reply:
> can you remember why you wanted to do this?

Not really, it was a while ago. :-)

Go ahead and remove this part if you like. Thanks!

Reply:
Done - I've kept the timestamp and moved the level_pfx back where it should be.